packages/styles/src/components/_trigger.scss

```diff
@@ -0,0 +1,38 @@
+$base-class: 'lc-trigger';
```

Comment:
`Trigger` is too much generic name for me. That's why I've suggested `lc-picker-trigger`.